ladybug/windrose.py

```diff
@@ -326,15 +326,33 @@ def histogram_data(self):
 
     @property
     def prevailing_direction(self):
-        """Get the predominant direction of the wind values.
-
-        In case of ties, this property will return the first direction value in the
-        ordered direction values.
+        """Get a tuple of the predominant directions of the wind values.
         """
+
         if self._prevailing_direction is None:
-            dirvals = sorted(self.direction_values)
-            self._prevailing_direction = dirvals[dirvals.index(max(dirvals))]
-        return self._prevailing_direction % 360.0
+            # group by frequency
+            groups = {}
+            for dirval in self.direction_values:
+                dirval = int(dirval)
+                if dirval not in groups:
+                    groups[dirval] = 1
+                else:
+                    groups[dirval] += 1
```

Comment:
I feel like this is really going to backfire on us if we ever have an EPW where the wind direction doesn't fall into 10-degree increments. Can we use the binned `_histogram_data` and the user-input `number_of_directions` for this case instead?